server/app/assets/javascripts/main.ts

```diff
@@ -162,6 +168,70 @@ function removeExistingEnumeratorField(event: Event) {
   enumeratorFieldDiv.appendChild(deletedEntityInput)
 }
 
+/** Add listeners to all enumerator inputs to update validation on changes. */
+function addEnumeratorListeners() {
+  // Assumption: There is only ever zero or one enumerators per block.
+  const enumeratorQuestion = document.querySelector('.cf-question-enumerator')
+  if (enumeratorQuestion) {
```

Comment:
suggestion (non-blocking): making this an early return would reduce the level of indentation for the rest of function
```suggestion
  if (!enumeratorQuestion) {
    return;
  }
```

Reply:
Done.